dev/com.ibm.ws.javaee.persistence.2.2/bnd.bnd

```diff
@@ -0,0 +1,43 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: com.ibm.ws.javaee.persistence.2.2; singleton:=true
+Bundle-Activator: com.ibm.ws.javaee.persistence.internal.HybridPersistenceActivator
+
+Import-Package: \
+  org.osgi.framework;version="[1.6,2)", \
+  org.osgi.util.tracker;version="[1.5,2)", \
+  javax.sql; resolution:=optional
+   
+# the jaxb feature will bring geronimo.osgi.registry to liberty, and the jpa api bundle
+# tracks the ProviderRegistry service, so we need dynamicly import the package.
+DynamicImport-Package: \
+  org.apache.geronimo.osgi.registry.api
+
+Export-Package: javax.persistence.*; jpa=2.2; consumer=persistenceService; mandatory:=consumer; -split-package:=first
+
+Private-Package: com.ibm.ws.javaee.persistence.internal, \
+   org.apache.geronimo.osgi.locator, \
+   org.apache.geronimo.specs.jpa
+
+Include-Resource: \
+   @${repo;javax.persistence:javax.persistence-api;2.2}!/about.html
+
+instrument.disabled: true
+
+-buildpath: \
+	javax.persistence:javax.persistence-api;version=2.2,\
+	com.ibm.websphere.org.osgi.core;version=latest,\
+	com.ibm.websphere.org.osgi.service.component;version=latest,\
+	com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
+	com.ibm.websphere.javaee.persistence.2.0;version=latest
```

Comment:
why jpa 2.0 on the buildpath?

Reply:
It looks like it's there because the HybridPersistenceActivator extends a class from it:

**import org.apache.geronimo.specs.jpa.PersistenceActivator;**

public class HybridPersistenceActivator extends **PersistenceActivator** {